docs/deputy-reporting-openapi-v1.yml

```diff
@@ -612,6 +520,22 @@ components:
       type: integer
       format: int32
       example: 12345
+    file:
+      type: object
+      required:
+        - name
+        - mimetype
+        - source
+      properties:
+        name:
+          type: string
+          example: Report_1234567T_2018_2019_11111.pdf
+        mimetype:
+          type: string
+          example: application/pdf
+        source:
+          type: string
+          description: The base64 encoded file
```

Comment:
Not sure if it will add much value or just muddy the water but there's a `byte` type you can specify for the format of string types to specify base64 encoded - https://swagger.io/docs/specification/data-models/data-types/

Reply:
Nice catch - done!